src/test/java/SeperateIntegerTest.java

```diff
@@ -0,0 +1,16 @@
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class SeperateIntegerTest {
+    @Test
+    void seperate()
+    {
+        String source = "1,2";
+        String[] splits = source.split(",");
+
+        assertThat(splits).contains("1");
+        assertThat(splits).contains("2");
+        assertThat(splits).containsExactly("1", "2");
```

Comment:
```suggestion
        assertThat(splits).containsExactly("1", "2");
```
`containsExactly`가 이미 포함여부를 확인하여서 해당 assertion은 제거해도 될 것 같아요 :)